registration/forms.py

```diff
@@ -247,7 +247,6 @@ def create_user(self, application, commit=True):
         user = User(
             username=self.cleaned_data['username'],
             email=self.cleaned_data['email'],
-            is_active=False,
```

Comment:
This pull request basically does what Braden is suggesting, since we are
already using django-simple-email-confirmation to validate email addresses.